flow360/component/simulation/meshing_param/face_params.py

```diff
@@ -2,12 +2,39 @@
 
 import pydantic as pd
 
-from flow360.component.simulation.entities_base import EntitiesBase
+from flow360.component.simulation.framework.base_model import Flow360BaseModel
+from flow360.component.simulation.framework.entity_base import EntityList
+from flow360.component.simulation.primitives import Face
 
+"""
+Meshing settings that applies to faces.
+"""
+
+
+class FaceRefinement(Flow360BaseModel):
+    """
+    These affects surface meshing.
+    """
+
+    entities: EntityList[Face] = pd.Field(alias="faces")
+    max_edge_length: LengthType.PositiveFloat = pd.Field(
+        description="Local maximum edge length for surface cells."
+    )
```

Comment:
why `Face` not `Surface`?

Reply:
Because meshing parameter are applied to faces which may or may not align with boundary surface definition. These are two independent ways of grouping small patches.

Reply:
ok, I see. We can keep it like this now, but I think this distinction is not required when we introduced entities. It was supposed to solve this problem.

Reply:
Then the entities corresponds to the patches which are the smallest unit. Problem is we do not have name for patches so the entities wildcard matching does not work.

I think we only have some sort of IDs for the patches (resource: @rundaji). Currently all operations are done either through Face (when you specify meshing parameter) or Surface/Group (when you specify boundary conditions).

Reply:
As discussed let's push the unification for surface and face.